src/java/htsjdk/samtools/reference/AbstractFastaSequenceFile.java

```diff
@@ -69,7 +73,38 @@
         }
     }
 
+    /**
+     * Finds and loads the sequence file dictionary.
+     * @param file Fasta file to read.  Also acts as a prefix for supporting files.
+     */
+    AbstractFastaSequenceFile(final Path file) {
```

Comment:
It seems like this is essentially a copy/paste of the other constructor.  Can the file constructor delegate to this one instead?